Apps/WebClient/src/ClientApp/src/components/report/header.vue

```diff
@@ -26,6 +27,8 @@ export default class ReportHeaderComponent extends Vue {
     private lastName = "";
     private phn = "";
     private dateOfBirth = "";
+    private retryCount = 2;
+    private eventBus = EventBus;
 
     private fetchPatientData() {
```

Comment:
good point, not sure why we leave the fetchPatientData here in the header instead of moving it to the report.vue though. as Patient data is required and displays in all the reports, I think it is better to move it to the report.vue as you suggest.